rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -748,16 +751,23 @@ public URI enlistCompensator(URI uri, Long timelimit, String linkHeader, StringB
 
         try {
             client = getClient();
+
             try {
+                MultivaluedMap<String, Object> headers = new MultivaluedHashMap<>();
```

Comment:
I'll still approve but I did mention the missing blank line on my first review.

Reply:
Done, thanks